include/swift/AST/AutoDiff.h

```diff
@@ -492,29 +492,39 @@ template<> struct DenseMapInfo<AutoDiffConfig> {
   static AutoDiffConfig getEmptyKey() {
     auto *ptr = llvm::DenseMapInfo<void *>::getEmptyKey();
     return {static_cast<IndexSubset *>(ptr), static_cast<IndexSubset *>(ptr),
-            DenseMapInfo<GenericSignature>::getEmptyKey()};
+            nullptr};
```

Comment:
For future reference: I wonder what errors arise from using `DenseMapInfo<GenericSignature>::getEmptyKey()` here?